angles/src/angles/__init__.py

```diff
@@ -0,0 +1,179 @@
+#*********************************************************************
+# Software License Agreement (BSD License)
+#
+#  Copyright (c) 2008, Willow Garage, Inc.
```

Comment:
Copied from here: https://github.com/ros/angles/blob/master/angles/include/angles/angles.h

What is the policy for updating Willow Garage copyrights? Nav stack is full of them.